MetaMask/metamask-mobile
# feat: upgrade assets controllers to version 43

## **Description**

Upgrades the assets controllers to versiom 43, which allows polling for token detection and token lists across chains.

It also lets us remove the patch for staking.

## **Related issues**

Depends on: https://github.com/MetaMask/metamask-mobile/pull/12270

## **Manual testing steps**

With PORTFOLIO_VIEW=false and PORTFOLIO_VIEW=true:

1. Onboard
2. Verify erc20 tokens are detected correctly
3. Switch chains
4. Verify they're detected correctly on the new chain



## **Screenshots/Recordings**

No visual changes

### **Before**



### **After**


https://github.com/user-attachments/assets/709ca6ba-69b8-4ec8-8fce-d358c7f39b27



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
